python/hail/methods/statgen.py

```diff
@@ -1689,57 +1676,119 @@ def pc_relate(dataset, k, maf, block_size=512, min_kinship=-float("inf"), statis
        PCRelate are often too noisy to reliably distinguish these pairs from
        higher-degree-relative-pairs or unrelated pairs.
 
+    Note that :math:`g_{is}` is the number of alternate alleles. Hence, for
+    multi-allelic variants, a value of 2 may indicate two distinct alternative
+    alleles rather than a homozygous variant genotype. To enforce the latter,
+    either filter or split multi-allelic variants first.
+
+    The resulting table has the first 3, 4, 5, or 6 fields below, depending on
+    the `statistics` parameter:
+
+     - `i` (``col_key.dtype``) -- First sample. (key field)
+     - `j` (``col_key.dtype``) -- Second sample. (key field)
+     - `kin` (:py:data:`.tfloat64`) -- Kinship estimate, :math:`\\widehat{\\phi_{ij}}`.
+     - `k2` (:py:data:`.tfloat64`) -- IBD2 estimate, :math:`\\widehat{k^{(2)}_{ij}}`.
+     - `k0` (:py:data:`.tfloat64`) -- IBD1 estimate, :math:`\\widehat{k^{(0)}_{ij}}`.
+     - `k1` (:py:data:`.tfloat64`) -- IBD0 estimate, :math:`\\widehat{k^{(1)}_{ij}}`.
```

Comment:
switch "IBD1 estimate" and "IBD0 estimate"

Reply:
It's a bit weird that we now have three of these referring to the mathematical notation and one of them referring to the english language description, but I don't really care much.

Reply:
updated